server/meshmodel/argo-cd/relationships/hierarchial_parent.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
@aniskhalfallah, this is good. Which aspect of an ApplicationSet and an Application need to be patched (reconfigured) when this relationship is realized? How does this relationship affect the configuration of one vs. the other?